addons/web/static/src/views/relational_model/relational_model.js

```diff
@@ -146,6 +146,66 @@ export class RelationalModel extends Model {
     // Protected
     // -------------------------------------------------------------------------
 
+    _applyProperties(data, config) {
+        for (const record of data.records) {
+            for (const fieldName in record) {
+                const field = config.fields[fieldName];
+                if (field && field.type === "properties") {
+                    const parent = record[field.definition_record];
+                    const relatedPropertyField = {
+                        fieldName,
+                    };
+                    if (parent) {
+                        relatedPropertyField.id = parent.id;
+                        relatedPropertyField.displayName = parent.display_name;
+                    }
+                    for (const property of record[fieldName]) {
+                        const propertyFieldName = `${fieldName}.${property.name}`;
+                        if (!config.fields[propertyFieldName]) {
+                            config.fields[propertyFieldName] = {
+                                ...property,
+                                name: propertyFieldName,
+                                relatedPropertyField,
+                                propertyName: property.name,
+                                relation: property.comodel,
+                            };
+                            config.activeFields[propertyFieldName] = {
```

Comment:
maybe considerer having a function in relational_model/utils.js that generates the activeField object from a property (basically lines 172-205)